cognite/client/data_classes/datapoints_subscriptions.py

```diff
@@ -317,35 +323,25 @@ class SubscriptionTimeSeriesUpdate:
     @classmethod
     def load(cls, data: dict[str, Any]) -> SubscriptionTimeSeriesUpdate:
         return cls(
-            added=[TimeSeriesID._load(added) for added in data.get("added", [])],
-            removed=[TimeSeriesID._load(added) for added in data.get("removed", [])],
+            added=[TimeSeriesID._load(ts_id) for ts_id in data.get("added", [])],
+            removed=[TimeSeriesID._load(ts_id) for ts_id in data.get("removed", [])],
         )
 
-    def dump(self, camel_case: bool = True) -> dict[str, Any]:
-        resource: dict[str, Any] = {}
-        resource["added"] = [id_.dump() for id_ in self.added]
-        resource["removed"] = [id_.dump() for id_ in self.removed]
-        return resource
+    def dump(self, camel_case: bool = True) -> dict[str, list[dict[str, Any]]]:
+        return {
+            "added": [ts_id.dump() for ts_id in self.added],
+            "removed": [ts_id.dump() for ts_id in self.removed],
+        }
 
 
 @dataclass
 class DatapointSubscriptionPartition:
     index: int
     cursor: str | None = None
 
-    @classmethod
-    def create(
-        cls, data: tuple[int, str | None] | int | DatapointSubscriptionPartition
-    ) -> DatapointSubscriptionPartition:
-        if isinstance(data, DatapointSubscriptionPartition):
-            return data
-        if isinstance(data, tuple):
-            return cls(*data)
-        return cls(data)
-
     @classmethod
     def load(cls, data: dict[str, Any]) -> DatapointSubscriptionPartition:
-        return cls(index=data["index"], cursor=data.get("cursor") or data.get("nextCursor"))
+        return cls(index=data["index"], cursor=data.get("nextCursor") or data.get("cursor"))
```

Comment:
Why are you getting both `nextCursor` and `cursor`? Shouldn't `nextCursor` be sufficient?

Reply:
This class is "misused" also for sending partitions with index and cursor

Reply:
ah, alright :)